client/jetpack-cloud/sections/partner-portal/primary/payment-method-list/index.tsx

```diff
@@ -0,0 +1,123 @@
+import DocumentHead from 'calypso/components/data/document-head';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+import { Card } from '@automattic/components';
+import { getAllStoredCards } from 'calypso/state/stored-cards/selectors';
+import QueryStoredCards from 'calypso/components/data/query-stored-cards';
+import {
+	VisaLogo,
+	MastercardLogo,
+	AmexLogo,
+} from 'calypso/my-sites/checkout/composite-checkout/components/payment-logos';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover/menu-item';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function StoredCard( props ): ReactElement {
+	let logo;
+	const translate = useTranslate();
+
+	if ( 'visa' === props.card.card_type ) {
+		logo = <VisaLogo />;
+	} else if ( 'mastercard' === props.card.card_type ) {
+		logo = <MastercardLogo />;
+	} else if ( 'amex' === props.card.card_type ) {
+		logo = <AmexLogo />;
+	}
+
+	const expiry = new Date( props.card.expiry );
+	const month = expiry.getMonth();
+	const year = expiry.getFullYear().toString().substr( 2, 2 );
+
+	return (
+		<div className="payment-method__stored-card">
+			<div className="payment-method__stored-card-header">
+				<div className="payment-method__stored-card-header-left">{ logo }</div>
+
+				<div className="payment-method__stored-card-header-right">
+					<Gridicon icon="checkmark" size={ 18 } />
+					<div className="payment-method__stored-card-primary">
+						<span className="payment-method__stored-card-primary-text">
+							{ translate( 'Primary' ) }
+						</span>
+					</div>
+					<EllipsisMenu className="payment-method__stored-card-ellipsis" position="bottom">
+						<PopoverMenuItem>{ translate( 'Make primary' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Update details' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Delete card' ) }</PopoverMenuItem>
+					</EllipsisMenu>
+				</div>
+			</div>
+			<div className="payment-method__stored-card-footer">
+				<div className="payment-method__stored-card-footer-left">
+					<div className="payment-method__stored-card-name">{ props.card.name }</div>
+					<div className="payment-method__stored-card-number">
+						**** **** **** { props.card.card }
+					</div>
+				</div>
+
+				<div className="payment-method__stored-card-footer-right">
+					<div className="payment-method__stored-card-expiry">{ `${ month }/${ year }` }</div>
+				</div>
+			</div>
+		</div>
+	);
+}
+
+function AddStoredCard(): ReactElement {
+	const translate = useTranslate();
+
+	return (
+		<div
+			className="payment-method__add-card"
+			onClick={ () => page( '/partner-portal/payment-method/add' ) }
+		>
+			<div className="payment-method__add-card-content">
+				<CardHeading className="payment-method__add-card-text" size={ 18 }>
+					{ translate( 'Add new credit card' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Add a new credit card' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-payment)